ocs_ci/ocs/resources/pod.py

```diff
@@ -2074,6 +2076,23 @@ def wait_for_storage_pods(timeout=200):
         helpers.wait_for_resource_state(resource=pod_obj, state=state, timeout=timeout)
 
 
+def wait_for_noobaa_pods(timeout=300):
+    """
+    Wait until all the noobaa pods have reached status RUNNING
+
+    Args:
+        timeout (int): Timeout in seconds
+
+    """
+
+    all_pod_objs = get_noobaa_pods()
```

Comment:
IIUC, this will return the pods with noobaa label that are available then. 
Shouldn't we ensure that the running pods are the ones we expect to run and not the ones available at a point in time?